tests/api_test.py

```diff
@@ -433,7 +433,7 @@ def for_each_tripple_func(src: ScAddr, edge: ScAddr, trg: ScAddr):
         )
         self.get_server_message('{"errors": [], "id": 1, "event": false, "status": true, "payload": ' + payload + "}")
         templ = ScTemplate()
-        templ.triple([ScAddr(0), "_class_node"], sc_types.EDGE_ACCESS_VAR_POS_PERM, ScAddr(0))
+        templ.triple((ScAddr(0), "_class_node"), sc_types.EDGE_ACCESS_VAR_POS_PERM, ScAddr(0))
```

Comment:
Can you now you square brackets?

Reply:
Yes, but with deprecated warning

Reply:
How long we should support deprecated methods? As I understand, we should remove every deprecated method in a next complete version, like 0.3.0? 
So, in the 0.2.5 release we will have bunch of deprecated methods with new ones. Should we discuss it in a separate discussion?

Reply:
I decided so myself. I thinks it would be great

Reply:
I added deprecated tests